content/v2/dns-analytics.markdown

```diff
@@ -80,23 +80,23 @@ Pagination is supported in this endpoint:
 ~~~
 curl  -H 'Authorization: Bearer <token>' \
       -H 'Accept: application/json' \
-      https://api.dnsimple.com/v2/dns_analytics
+      https://api.dnsimple.com/v2/1385/dns_analytics
```

Comment:
```suggestion
      https://api.dnsimple.com/v2/<account_id>/dns_analytics
```
☝️ shouldn't we parametrize this like we do with `token`?

Reply:
I've put the same thing we're using in other dev docs...

Reply:
then I'll 🤐